### PR TITLE
[FastPR][Fsi] Minors in Randomized MVQN defaults and documentation

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
Extra documentation and minor changes in the defaults for the RandomiZed MVQN convergence accelerator. The new settings are proved to be more robust than previous ones.

### CODE REVIEW

**Comment by rubenzorrilla** [comment]
Closing as it seems that I messed two branches

File: kratos/operations/operation.h [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Carlos Roig
+//                   Ruben Zorrilla
+//
+
+#pragma once
```
Comment (RiccardoRossi) [inline kratos/operations/operation.h]: @rubenzorrilla is this expected to go in as a part of this PR?

**Review by RiccardoRossi — COMMENTED** [review]
@rubenzorrillayou are committing multistage stuff which does not belong to this PR, i am right?

File: kratos/operations/operation.h (hunk 1 above)
Comment (rubenzorrilla) [reply kratos/operations/operation.h]: Nope. These accidentally sneaked in (this is why I closed this one). See my comment above. I'm creating another one with the MVQN-RZ changes only.

**Pull request closed without merging**